guides/common/assembly_configuring-satellite-custom-server-certificate.adoc

```diff
@@ -6,14 +6,29 @@ ifdef::context[:parent-context: {context}]
 By default, {ProjectName} uses a self-signed SSL certificate to enable encrypted communications between {ProjectServer}, external {SmartProxyServers}, and all hosts.
 If you cannot use a {Project} self-signed certificate, you can configure {ProjectServer} to use an SSL certificate signed by an external Certificate Authority.
 
+When you configure {ProductName} with custom certificates, you must follow the requirements mentioned below:
+
+* You must use the Privacy-Enhanced Mail (PEM) encoding for the SSL certificates.
+* You cannot use the same certificate for both {ProjectServer} and {SmartProxyServer}.
+* The same Certificate Authority must sign certificates for {ProjectServer} and {SmartProxyServer}.
+* Certificate cannot also be a CA certificate.
+* Certificate must include a Subject Alt Name (SAN) entry that matches the Common Name (CN).
+* Certificate must be allowed for Key Encipherment via a Key Usage extension.
+* Certificate cannot have a shortname as the Common Name (CN).
+* A passphrase cannot be set for the private key.
```

Comment:
I'd use "must not" instead of "cannot", because we want to forbid these things, even though they are technically possible.

@bangelic @asteflova Can you give your opinions on this, please?

Reply:
I agree to `must not` :+1: